HW2-W2-D3/Random_Images/app.js

```diff
@@ -0,0 +1,25 @@
+// this is Js file for Random_Images  .
+
+
+//  clicks button #randomize
+document.querySelector('#randomize').addEventListener('click',function(){
+
+    let imgWidth = document.querySelector('.width').value;
+    let imgHeight= document.querySelector('.height').value;
+// Create an img element
+    let newImg = document.createElement('img');
+
+    newImg.setAttribute('src','https://source.unsplash.com/random/160x900');
+
+// Bonus 1    
+    newImg.setAttribute('width',imgWidth);
+    newImg.setAttribute('height',imgHeight);
+
+    
+    //newImg.style.width = imgWidth ;
```

Comment:
to make the project call a different sized picture 
make the height and width  a variable.

and let the URL similar to this 

```
let url = ("https://source.unsplash.com/random/" + width.value + "x" + height.value);
```